recipe/meta.yaml

```
@@ -29,15 +29,15 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
-    - {{ cdt('libx11-devel') }}          # [linux]
-    - {{ cdt('libxcb') }}                # [linux]
-    - {{ cdt('libxau') }}                # [linux]
     - make                               # [linux]
     - patch                              # [osx and x86_64]
     - m2-patch                           # [win]
   host:
     - zlib {{ zlib }}
+    - xorg-xorgproto
+    - xorg-libx11  # [linux]
+    - xorg-libxau
+    - libxcb  # [linux]
```

```suggestion
    - xorg-xorgproto. # [linux]
    - xorg-libx11  # [linux]
    - xorg-libxau. # [linux]
    - libxcb  # [linux]
```
Wouldn't these all stay linux-only deps?

---

You're right. I forgot to constrain them. Other packages in the build order may have the same issue. I'll fix it later